util/config/gather-cray-prgenv-arguments.bash

```diff
@@ -27,7 +27,14 @@ if [[ $FAIL == 1 || -z $2 || -z $3 || -z $4 ]]; then
     exit 1
 fi
 
-# Set up the environment to amke the proper libraries and include
+# Try loading PrgEnv-gnu if no module is loaded
+module list 2>&1 | grep -q PrgEnv
```

Comment:
Typically we through --terse or -t to the `module list` command in order to output results on separate lines and without numbering prefixes. I'm not sure if it matters here, but it couldn't hurt